experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl

```diff
@@ -1245,7 +1245,7 @@ function euler_characteristic(m::AbstractFTheoryModel; check::Bool = true)
 
   # Check if the answer is known
   if has_attribute(m, :euler_characteristic)
-    return get_attribute(m, :euler_characteristic)::CohomologyClass
+    return get_attribute(m, :euler_characteristic)
```

Comment:
Why does this get removed here?

Reply:
The type is wrong. It should be changed to `::Int` (not dropped).

Reply:
Or another type for integers. Euler characteristics can be large. Maybe `ZZRingElem` (or alike?) makes more sense? Maybe you know @lgoettgens ?

Reply:
You should annotate here what really is saved in the attributes. Other functions for things like this take a IntegerUnion subtype (so eg Int, Int128, ZZRingElem) as the first argument to determine the return type. If something doesn't fit, the conversion in there will throw some error

Reply:
Ok. On 2nd thought, I think `Int` (`Int64`) is large enough for now. So a change to `::Int` should suffice (at least for now).

Reply:
I have now changed it to ::Int

Reply:
I also had to convert it to ::Int as initially it was of type QQFieldElem.